### ISSUE TITLE
[Proxy colonies M0] feat: prototyping and setting up the dev environment for the proxy colonies

### ISSUE THREAD

**Issue by bassgeta** [JoinColony/colonyCDapp#3408]
### Story

Before we start work on other milestones, we need to prototype creating a proxy colony on a supported chain and set up the development environment.

### Description

The scope of this PR is just a temporary button saying "Create new proxy colony" which should call [this contract](https://github.com/JoinColony/colonyNetwork/blob/428de9345aef7ed39a928cad7b4d5c483712e96e/contracts/colony/Colony.sol#L327). 
After this contract is done, create a new `ProxyColony` entry in the database and list it in a simple span.
Q: calling this contract doesn't seem to emit any event, and the tests create contracts which get data filled in. Can the block ingestor catch this via some events or do we need to call amplify from the CDapp?

### This issue is complete when...

- [ ] The dev environment starts up with multiple chains
- [ ] You can create a transaction creating a proxy colony on a different chain
- [ ] After it's created we note that in the database and we can display it in the CDApp

**Comment by rdig** [comment]
This is done right ?

**Comment by rdig** [comment]
Right, my questions was more like, if everything was merged, do we still need this issue open :)
